### PR TITLE
Signup template

### PR DESCRIPTION
**Pull request by hkcj**

### CODE REVIEW

**Comment by hkcj** [comment]
Just the template.
should i link routes and push again?

File: src/signup/SignUp.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  form: {
+    width: "100%", // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+export default function SignUp() {
```
Comment (zt1983811) [inline src/signup/SignUp.tsx]: Can we do arrow function like what we used to do please?
`const SignUp = () => {}`

File: src/signup/SignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
+import React from "react";
```
Comment (zt1983811) [inline src/signup/SignUp.tsx]: Can we move this file under landing/modules/component ?

File: src/landing/moudles/components/EmailText.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import TextField from "@material-ui/core/TextField";
+
+const Emailtext = () => {
```
Comment (zt1983811) [inline src/landing/moudles/components/EmailText.tsx:3]: Question: if you have already extracted email as a separate component, can we reuse it also in SignUp component ?

File: src/landing/moudles/components/SigninButton.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import { makeStyles } from "@material-ui/core/styles";
+import { AdminBasePath } from "../../../admin/moudles/Routes";
+
+const useStyles = makeStyles((theme) => ({
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+const Signinbutton = () => {
```
Comment (zt1983811) [inline src/landing/moudles/components/SigninButton.tsx:11]: Same question as email component , can we pass in text to make it reuse button?

File: src/landing/moudles/components/PasswordText.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import Textfield from "@material-ui/core/TextField";
+
+const PasswordText = () => {
```
Comment (zt1983811) [inline src/landing/moudles/components/PasswordText.tsx:3]: Same question as email component

**Comment by hkcj** [comment]
Yes, we can reuse the components

**Pull request closed without merging**